pkg/artifactory/resource_xray_watch.go

```diff
@@ -350,7 +350,7 @@ func resourceXrayWatchRead(d *schema.ResourceData, m interface{}) error {
 
 func resourceXrayWatchUpdate(d *schema.ResourceData, m interface{}) error {
 	watch := expandWatch(d)
```

Comment:
If you look at what I did elsewhere, I put `this must be a pointer` - hey wait: you're fixing this in deprecated xray code. Did you get an internal customer ticket?